### PR TITLE
1.23.4 with aiven changes

### PR DESCRIPTION
**Pull request by mhoffm-aiven**
We were already running 1.23.4 on branch `origin/a-safa-1.23.4`; I based cross cluster replication on top of that and want to merge it back into 1.23.4-aiven so that we can use this branch for deployments.

### CODE REVIEW

File: AIVEN_CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Input Plugins
 
+### Elasticsearch
+
+* add cross cluster replication metrics ( they dont work for elasticsearch but its a first step unil we have an opensearch plugin )
```
Comment (safa-topal) [inline AIVEN_CHANGES.md:7]: > they dont work for elasticsearch but its a first step unil we have an opensearch plugin

I'm not sure what this means, this should be expanded a bit more for clarity. Also typo in `unil`

File: plugins/inputs/mysql/README.md [hunk 2]
```diff
@@ -126,6 +126,9 @@ in mySQL configuration. See the performance schema [quick start][quick-start].
   # tls_key = "/etc/telegraf/key.pem"
   ## Use TLS but skip chain & host verification
   # insecure_skip_verify = false
+
+  ## Aggregate table_io_waits
+  # aggregate_table_io_waits = true
```
Comment (safa-topal) [inline plugins/inputs/mysql/README.md:131]: What's the reasoning for this change? It doesn't seem relevant to CCR

**Review by safa-topal — CHANGES_REQUESTED** [review]
No big change requests, lgtm for the most part. Left a request to rephrase a sentence and also a question about seemingly-irrelevant change in the commit

File: AIVEN_CHANGES.md (hunk 1 above)
Comment (mhoffm-aiven) [reply AIVEN_CHANGES.md:7]: elasticsearch does not have this plugin api. It will only work for opensearch is what i mean

File: plugins/inputs/mysql/README.md (hunk 2 above)
Comment (mhoffm-aiven) [reply plugins/inputs/mysql/README.md:131]: its not in the CCR commit, its from another commit, just included in the PR

Comment (mhoffm-aiven) [reply plugins/inputs/mysql/README.md:131]: I want to base all those commits on top of "1.23.4-aiven"

File: AIVEN_CHANGES.md (hunk 1 above)
Comment (mhoffm-aiven) [reply AIVEN_CHANGES.md]: fixed typo

File: plugins/inputs/mysql/README.md (hunk 2 above)
Comment (mhoffm-aiven) [reply plugins/inputs/mysql/README.md:131]: https://github.com/aiven/telegraf/pull/76/commits/b767cc6dd22a229456cd1ae43a2ffcd17de3981c this is the only new commit

**Review by axeoman — APPROVED** [review]
